google/devtools/clouddebugger/v2/clouddebugger_gapic.yaml

```diff
@@ -0,0 +1,167 @@
+type: com.google.api.codegen.ConfigProto
+language_settings:
+  java:
+    package_name: com.google.cloud.devtools.clouddebugger.spi.v2
+  python:
+    package_name: google.cloud.devtools.clouddebugger.v2
```

Comment:
Good catch @jmuk! The Python namespace is `google.cloud.grpc.{api}.{version}` and `google.cloud.gapic.{api}.{version}`.